Remotion/Development/UnitTesting.Compilation/Roslyn/CSharpRoslynCodeProvider.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) RUBICON IT GmbH, www.rubicon.eu
```

Comment:
Please add this to PackageInfo.ps1 (see other embedded sources) and to dependency docs in wiki